tests/gradients/finite_diff/test_spsa_gradient_shot_vec.py

```diff
@@ -30,6 +30,7 @@
 
 default_shot_vector = (1000, 2000, 3000)
 many_shots_shot_vector = tuple([100000] * 3)
+np.random.seed(42)
```

Comment:
Unrelated change? Fixing the seed on the file level might not be optimal with distributed testing?

Reply:
```suggestion
```